cli.js

```diff
@@ -92,33 +95,33 @@ st.once('testserver', function (server) {
 
 	setState('download');
 	var ping = Math.round(server.bestPing);
-	stats.ping = (cli.flags.json) ? ping : chalk.cyan(ping + chalk.dim(' ms'));
+	stats.ping = cli.flags.json ? ping : chalk.cyan(ping + chalk.dim(' ms'));
 });
 
 st.on('downloadspeedprogress', function (speed) {
 	if (state === 'download' && cli.flags.json !== true) {
-		var download = roundTo(speed, speed > 10 ? 0 : 1);
-		stats.download = chalk.yellow(download + chalk.dim(' Mbps'));
+		var download = roundTo(speed * multiplier, speed > 10 ? 0 : 1);
```

Comment:
Those are different. The original speed is in bits per second. If the speed is greater then 10 Mbps (for instance 14.3 Mbps), it will be printed as `14 Mbps`.

If you use bytes, it will still check if the bits per second is greater then 10 to round down the number of bytes per second, which is not correct.